Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/ConstantForceField.inl

```diff
@@ -40,179 +40,146 @@ template<class DataTypes>
 ConstantForceField<DataTypes>::ConstantForceField()
     : d_indices(initData(&d_indices, "indices", "indices where the forces are applied"))
     , d_indexFromEnd(initData(&d_indexFromEnd,false,"indexFromEnd", "Concerned DOFs indices are numbered from the end of the MState DOFs vector. (default=false)"))
-    , d_forces(initData(&d_forces, "forces", "applied forces at each point"))
-    , d_force(initData(&d_force, Deriv{}, "force", "applied force to all points if forces attribute is not specified"))
-    , d_totalForce(initData(&d_totalForce, "totalForce", "total force for all points, will be distributed uniformly over points"))
+    , d_inputForces(initData(&d_inputForces, "forces", "applied forces at each point"))
+    , d_inputForce(initData(&d_inputForce, Deriv{}, "force", "applied force to all points if forces attribute is not specified"))
```

Comment:
I recommand removing inputForce because there are multiple better ways to do the same results:
- use the python syntax [1,2,3]*100 (to make [[1,2,3],[1,2,3], [1,2,3]... 100 times) 
- use a DataEngine with a single  input force and with an integer with an output an array of 'n' items. 
- use the rules which consist in duplicating the last entry of an array to match the actual size... so setting inputForces = [ [1,2,3]] end in having the [1,2,3] repeated to mach the size of output.
-

Reply:
This is why I proposed two other alternative to the python syntax :)